### ISSUE TITLE
jsx-wrap-multiline issue with trailing-comma

### ISSUE THREAD

**Issue by SimonSchick** [palantir/tslint-react#214]
### Bug Report

- __tslint-react version__: 4.0.0
- __tslint version__: 5.14.0
- __typescript version__: 3.4
- __Running TSLint via__: NodeJS v11.13

#### TypeScript code being linted

```ts
      group.push(
        <Button
          id={`assign${this.props.dealer.id}`}
          key='assign'
          onClick={() => this.props.openAssignTable()}
        >
          Assign Table
        </Button>,
      );
```

with `tslint.json` configuration:

```json
    "trailing-comma": [true, {
        "esSpecCompliant": true,
        "multiline": "always",
        "singleline": "never"
      }],
      "jsx-wrap-multiline": true

```

#### Actual behavior
Lints fine.

#### Expected behavior
When a trailing comma is added the rule complaints about not being wrapped.
While this is technically correct I think an exception should be made in the case of being the single argument with a trailing comma.

**Comment by adidahiya** [comment]
There are a lot of issues with this rule, see https://github.com/palantir/tslint-react/issues/79. My suggestion is to use prettier.

**Comment by SimonSchick** [comment]
Cool, I already disabled it, kinda sad tho.
